tests/file_adapter_tests.py

```diff
@@ -1,7 +1,7 @@
 """Unit Tests for File Adapter."""
 
 import unittest
-from file_adapter import FileAdapter
+from ..data.file_adapter import FileAdapter
```

Comment:
use __init__.py files and add the root of the repository in the sys.path in this file.